src/middlewares/validateRejectTrip.js

```diff
@@ -30,7 +30,7 @@ export const isTripExist = async (req, res, next) => {
       tripId
     }
   } = req;
-  const trip = await travelRequests.findOne({ where: { requestId: tripId } });
+  const trip = await travelRequests.findOne({ where: { id: tripId } });
```

Comment:
@blaise82 Why did you change ```requestId``` to ``id``

Reply:
it was to make sure that when someone fetches requests we can also get comments made to that request 

using table relationships
our tables had no relationships and sequelize  looks for the id to refer to

Reply:
@blaise82 Sure thing, I understand